doc/org/build/gameplay.md

```diff
@@ -0,0 +1,722 @@
+
+Assume the `MediaPrefab` template is initialized with `src` set to the url of a `gltf` file. In other words, we are trying to use the `MediaPrefab` load a `gltf` file that we can grab and interact with.
+
+We pass the `EntityDef` to `renderAsEntity`, an entity is returned synchronously, and execution continues as normal.
+
+When the `mediaLoading` system runs, it notices the new `MediaLoader` component and starts an asynchronous `coroutine` that determines the media type pointed to by the `src` property, downloads the `gltf` file and loads it as Three.js scene graph via a `GLTFLoader`. Finally, the loaded scene graph is passed into an `EntityDef`&rsquo;s `model` parameter:
+
+```typescript
+export function* loadModel(world: HubsWorld, src: string, contentType: string, useCache: boolean) {
+  const { scene, animations } = yield loadGLTFModel(src, contentType, useCache, null);
+  scene.animations = animations;
+  scene.mixer = new THREE.AnimationMixer(scene);
+  return renderAsEntity(world, <entity model={{ model: scene }} />);
+}
+```
+
+In other words, the asynchronous work of loading the model is done `before` an entity is created for the loaded `gltf`. Despite being called after some asynchronous work, the entity creation step that happens in `renderAsEntity` itself happens synchronously.
+
+The `model` inflator (`inflateModel`) is to `gltf` files as `renderAsEntity` is to `EntityDef` s. You will notice that `inflateModel` invokes other component inflators and adds many entities to the world to match the `Object3D` hierarchy that was provided within the `ModelParams`.
+
+This is what we mean when we say that loading from `gltf` files is &ldquo;equivalent&rdquo; to loading from `EntityDef` s.
+
+
+<a id="org8ca2b33"></a>
+
+### Common inflators, `jsxInflators`, and `gltfInflators`
+
+We can now explain why there are three collections of `inflators`:
+
+-   `commonInflators` are for components can be loaded (identically) whether they are defined in `EntityDef` s or `gltf` files.
+-   `jsxInflators` are for components that are specified in `EntityDef` s.
+-   `gltfInflators` are for components that are specified in `gltf` files.
+
+Notice that in the sentences above, we overload the word `components`. As we have shown, the data in `EntityDef` s or `gltf` files are not `bitECS` components, but need to be transformed (via `inflator` s) to their runtime formats (which is usually `bitECS` components).
+
+While it might be helpful to define a `new` word (like &ldquo;pre-components&rdquo;) to describe these data, we think this is overly complicated. From the perspective of the Blender add-on for example, its job is to add components (specifically, &ldquo;`MOZ_hubs_components`&rdquo;) to nodes in the `.blend` scene and exported `gltf` files.
+
+
+<a id="orgfcfa191"></a>
+
+### Entity `Ref` s and `__mhc_link_type` : `"node"`
+
+
+<a id="org7fe068b"></a>
+
+### Associating `Material` s (`eid2mat`)
+
```

Comment:
We have explained how entity and components are instantiated until here. Probably it is worth to also explain how they and their associated resources are/should be cleaned up?